lib/src/css_mirroring/README.md

```diff
@@ -0,0 +1,98 @@
+BidiCssGenerator generates a CSS which comprises of orientation neutral, orientation specific and flipped orientation
```

Comment:
I would move that file to the root of the project (next to README.md), name it BidirectionalCss.md, and link to it from the main README.md